_targets.R

```diff
@@ -59,6 +59,9 @@ mainstem_reaches_tidal <- c("2771_1","2769_1","2768_1","2767_1","2764_1","2762_1
 # Define the url for the NHGFv1 to NHDv2 crosswalk
 GFv1_NHDv2_xwalk_url <- "https://raw.githubusercontent.com/USGS-R/drb-network-prep/main/2_process/out/GFv1_NHDv2_xwalk.csv"
```

Comment:
```suggestion
GFv1_NHDv2_xwalk_url <- "https://raw.githubusercontent.com/USGS-R/drb-network-prep/c45f469098dfb245a2d686b807a58afa00b9d0b2/2_process/out/GFv1_NHDv2_xwalk.csv"
```
```suggestion
GFv1_NHDv2_xwalk_url <- "https://raw.githubusercontent.com/USGS-R/drb-network-prep/main/2_process/out/GFv1_NHDv2_xwalk.csv"
```

See comment below about changing these to a specific commit instead of using the latest on main